targets/mate

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+if [ "${TARGETNOINSTALL:-c}" = 'c' ]; then
+   if release -lt trusty; then
+       error 99 "Mate packages are not available for Ubuntu releases prior to trusty."
+   fi
+   if release -lt jessie; then
+       error 99 "Mate packages are not available for Debian releases prior to jessie."
+   fi
+   if release -lt sana; then
+       error 99 "Mate packages are not available for Kali Linux prior to sana."
+   fi
+fi
+
+REQUIRES='x11'
+DESCRIPTION='Installs the mate desktop environment. (Approx. 205MB)'
+HOSTBIN='startmate'
+CHROOTBIN='crouton-noroot startmate'
+. "${TARGETSDIR:="$PWD"}/common"
+
+EXTRAPACKAGES=
+
+if release -le precise ; then
+    install software-properties-common
+    sudo apt-add-repository -y "deb http://packages.mate-desktop.org/repo/ubuntu precise main"
+    sudo apt-get update
+fi
+
+if release -ge xenial ; then
+    install software-properties-common
+    sudo apt-add-repository -y ppa:ubuntu-mate-dev/xenial-mate
+    sudo apt-get update
+    EXTRAPACKAGES="${EXTRAPACKAGES} caja-actions ubuntu-mate-artwork"
+fi
+
+if release -eq trusty && [ "$ARCH" = 'amd64' -o "$ARCH" = 'i386' ]; then
+    install software-properties-common
+    sudo apt-add-repository -y ppa:ubuntu-mate-dev/ppa
```

Comment:
Yes, there are arm binaryies for trusty:
http://ppa.launchpad.net/ubuntu-mate-dev/trusty-mate/ubuntu/dists/trusty/main/

So that needs to be changed.  It shouldn't be only amd64 and i386.